<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppletIdentityClass</class>
 <widget class="QWidget" name="AppletIdentityClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="m_OnlineIdentityGroupBox">
     <property name="title">
      <string>Online Identity</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="m_UserNameLabel">
            <property name="text">
             <string>User Online Name: ( 4 to 23 characters )</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_OnlineNameEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="m_ApplyNameButton">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="m_MoodMessageLabel">
            <property name="text">
             <string>Mood Message: ( Up to 27 characters )</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_MoodMessageEdit">
            <property name="text">
             <string>Let's Communicate!</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="m_ApplyMoodButton">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Age:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_AgeEdit">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Preferred Content:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_ContentComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_ApplyAgeButton">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Gender:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_GenderComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Primary Language:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_LanguageComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_ApplyGenderButton">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_AboutMeGroupBox">
     <property name="title">
      <string>About Me Page</string>
     </property>
     <layout class="QVBoxLayout" name="DialogItemsGroupBox_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="PermissionWidget" name="m_PermissionWidget" native="true"/>
      </item>
      <item>
       <widget class="QLabel" name="m_StatusLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_ApplyAboutMeButton">
        <property name="text">
         <string>Apply About Me Changes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_GreetingLabel">
        <property name="text">
         <string>Greeting:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_GreetingEdit">
        <property name="text">
         <string>Hi There!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="aboutMeLabel">
        <property name="text">
         <string>About Me:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="m_AboutMeEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="plainText">
         <string>I'm Wonderfull!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_PictureOfMeLabel">
        <property name="text">
         <string>My Picture</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="m_BrowsePictureButton">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_TakeSnapshotButton">
          <property name="text">
           <string>Take Snapshot</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="m_PictureOfMeFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="favoriteWebsitesLabel">
        <property name="text">
         <string>My Favorite Websites</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_FavoriteWebsite1Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="m_FavoriteWebsite2Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="m_FavoriteWebsite3Edit">
        <property name="text">
         <string>http://www.gotvptop.com</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Donation Information</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="m_DonationEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PermissionWidget</class>
   <extends>QWidget</extends>
   <header>PermissionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
